hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2434,13 +2434,18 @@
     {
       name: chip_sw_keymgr_sideload_kmac_error
       desc: '''
-            Verify the effect of KMAC returning an error during keymgr sideload operation.
+            Verify the effect of KMAC returning an error during a keymgr operation.
 
-            - Follow the steps in `chip_sw_keymgr_sideload_kmac` test.
+            - Configure keymgr to enter any of the 3 working states.
+            - Issue a keymgr operations.
```

Comment:
super nit: we can drop the plural (`a keymgr operation.`)

Reply:
Fixed, thanks.